tools/build_sovereign_book.sh

```
@@ -0,0 +1,173 @@
+for file in $LISTED_FILES; do
+    if [ -f "$file" ]; then
+        target="$TEMP_DIR/$(basename "$file")"
+        # Flatten tables + strip sub-metadata blocks
+        pandoc "$file" --from markdown-yaml_metadata_block -t markdown-grid_tables+pipe_tables -o "$target"
+        PROCESSED_FILES="$PROCESSED_FILES $target"
+    fi
+done
+
+# --- [6. Build Engine (LuaLaTeX + Header Injection)] ---
+echo "🏗️  Building Sovereign Book for ${CURRENT_USER} via LuaLaTeX..."
+if pandoc $PROCESSED_FILES \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)